tag-publish-docker-image.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Ah und it's also a better idea to compute the docker variables via environment variables. Otherwise you will see the credentials of our docker account in the logs.